modules/gcp_bigquery_big_lake_table/main.tf

```diff
@@ -12,18 +12,34 @@ resource "google_bigquery_table" "google_bigquery_table" {
   dataset_id          = var.dataset_id
   deletion_protection = var.deletion_protection
   description         = var.description
-  schema              = var.schema
   table_id            = var.name
 
+  # Terraform will detect changes to this property made by BigQuery, but we'll ignore them using the `lifecycle` block.
+  schema = var.schema
+
   external_data_configuration {
     autodetect    = var.autodetect
     connection_id = var.connection_id
     source_format = var.source_format
     source_uris   = var.source_uris
 
+    # Use the exact same schema here. This one won't be changed by BigQuery, however Terraform will still detect the changes you make on purpose to this field.
+    schema = var.schema
```

Comment:
I saw in the document
```
Schema is disallowed for Google Cloud Bigtable, Cloud Datastore backups, Avro, Iceberg, ORC and Parquet formats.
```
Under the `external_data_configuration` all BigLake tables are reading Parquet files from GCS bucket, or Are you preparing this with the other scenario?

Reply:
Also from this in the top-level `schema`

> If you use external_data_configuration [documented below](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#nested_external_data_configuration) and do not set external_data_configuration.connection_id, schemas must be specified with external_data_configuration.schema. Otherwise, schemas must be specified with this top-level field.

We use `external_data_configuration.connection_id` for the BigLake connection. I think we can ignore this.

Reply:
Ah I need to update our example then, our example uses:

```
source_format = "NEWLINE_DELIMITED_JSON"
```